build/actions/wizard.js

```diff
@@ -0,0 +1,89 @@
+(function() {
+  var _, async, capitano, resin, visuals;
+
+  _ = require('lodash-contrib');
+
+  capitano = require('capitano');
+
+  visuals = require('resin-cli-visuals');
+
+  async = require('async');
+
+  resin = require('resin-sdk');
+
+  exports.wizard = {
+    signature: 'quickstart [name]',
+    description: 'getting started with resin.io',
+    help: 'Use this command to run a friendly wizard to get started with resin.io.\n\nThe wizard will guide you through:\n\n	- Create an application.\n	- Initialise an SDCard with the resin.io operating system.\n	- Associate an existing project directory with your resin.io application.\n	- Push your project to your devices.\n\nExamples:\n\n	$ sudo resin quickstart\n	$ sudo resin quickstart MyApp',
+    root: true,
+    permission: 'user',
+    action: function(params, options, done) {
+      return async.waterfall([
+        function(callback) {
+          if (params.name != null) {
+            return callback();
+          }
+          return async.waterfall([
+            function(callback) {
+              return resin.models.application.hasAny(callback);
+            }, function(hasAnyApplications, callback) {
+              if (!hasAnyApplications) {
+                return callback(null, null);
+              }
+              return async.waterfall([
+                function(callback) {
+                  return resin.models.application.getAll(callback);
+                }, function(applications, callback) {
+                  applications = _.pluck(applications, 'app_name');
+                  applications.unshift({
+                    name: 'Create a new application',
+                    value: null
+                  });
+                  return visuals.widgets.select('Select an application', applications, callback);
+                }
+              ], callback);
+            }, function(application, callback) {
+              if (application != null) {
+                return callback(null, application);
+              }
+              return visuals.widgets.ask('What\'ll be the name of the application?', null, function(error, applicationName) {
+                return capitano.run("app create " + applicationName, function(error) {
+                  if (error != null) {
+                    return callback(error);
+                  }
+                  return callback(null, applicationName);
+                });
+              });
+            }, function(applicationName, callback) {
+              params.name = applicationName;
+              return callback();
+            }
+          ], callback);
+        }, function(callback) {
+          return capitano.run("device init --application " + params.name, callback);
+        }, function(deviceName, callback) {
+          params.device = deviceName;
+          console.log('Please install SD card in Raspberry Pi, Connect it to network and power it up.');
```

Comment:
@shaunmulligan Nice catch. I think I copy pasted this from your Google Doc and didn't pay attention to the hardcoded device type. I'll fix now.